preact/components/recommendation/bundle/default/Variables.yaml

```diff
@@ -0,0 +1,18 @@
+# component descriptor file
+
+# custom component variables
+variables: [
+  {
+    name: 'addToCart',
+    message: 'Choose an addToCart method:',
+    type: 'list',
+    choices: ['shopify', 'magento2', 'bigcommerce', 'custom'],
+    choicesMap: {
+      shopify: "import { addToCart } from '@searchspring/snap-platforms/shopify';",
+      magento2: "import { addToCart } from '@searchspring/snap-platforms/magento2';",
+      bigcommerce: "import { addToCart } from '@searchspring/snap-platforms/bigcommerce';",
+      custom: "const addToCart = (e, items)=> console.log('need to add these to the platform cart', items);",
+    },
```

Comment:
Did you look into using the object type of 'choices' with name/value?
https://www.npmjs.com/package/inquirer#question:~:text=choices%3A%20(Array%7CFunction,a%20Separator.